src/main/java/Task.java

```diff
@@ -7,15 +7,15 @@ public Task(String description) {
         this.isDone = false;
     }
 
-    public String getDescription(){
+    public String getDescription() {
         return description;
     }
 
     public String getStatusIcon() {
         return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Perhaps you could consider saving the  "u/2713" as a const variable if there is a need to reuse it in the future?

Reply:
That's a good idea, thanks!